internal/observer/observer.go

```diff
@@ -94,11 +94,10 @@ type observer struct {
 	sink func(LoggedEntry) error
 }
 
-// New creates a new facility that buffers logs in memory (without any
-// encoding). It's particularly useful in tests, though it can serve a variety
-// of other purposes as well. This constructor returns the facility itself and
-// a function to retrieve the observed logs.
-func New(enab zapcore.LevelEnabler, sink func(LoggedEntry) error, withContext bool) zapcore.Facility {
+// New creates a new Core that buffers logs in memory (without any encoding).
+// It's particularly useful in tests, though it can serve a variety of other
```

Comment:
"thought i can serve a variety of other purposes" -- seems a little vague. If we don't have any other use cases to call out, is it worth having this comment?